config/rbac/leader_election_role_binding.yaml

```diff
@@ -1,12 +1,20 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: RoleBinding
 metadata:
-  name: leader-election-rolebinding
+  labels:
+    app.kubernetes.io/name: rolebinding
+    app.kubernetes.io/instance: kaoto-operator-leader-election-rolebinding
+    app.kubernetes.io/component: rbac
+    app.kubernetes.io/created-by: kaoto-operator
+    app.kubernetes.io/part-of: kaoto-operator
+    app.kubernetes.io/managed-by: kustomize
+  name: kaoto-operator-leader-election
+  namespace: kaoto-system
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: Role
-  name: leader-election-role
+  name: kaoto-operator-leader-election
 subjects:
 - kind: ServiceAccount
-  name: controller-manager
-  namespace: system
+  name: kaoto-operator
+  namespace: kaoto-system
```

Comment:
Does this imply a dedicated namespace for the kaoto-operator? Would the operator not be installed in the defauit OLM namespace, eg. `openshift-operators` on Openshift?

Reply:
In my understanding, this only impacts installation with kubeclt, when making a bundle, the namespace is stripped out and managed by OLM (AFAIK)